inst/pages/miaverse.qmd

```diff
@@ -245,12 +246,24 @@ the troubleshoot page
 [here](https://github.com/microbiome/OMA/blob/devel/PackageInstallations_Troubleshoots.qmd)
 and [@sec-support].
 
+## Interactive analysis {#sec-webapp}
+
+We also provide the Microbiome Analysis Dashboard (miaDash), a web app to
+interactively explore microbiome data through an intuitive Graphical User
+Interface (GUI). The app features a large part of the mia functionality and
+does not require any knowledge of R programming. This way, beginners can practice
+and learn the concepts of microbiome analysis while free from technical burdens.
+The app is available online at
+[this address](https://miadash-microbiome.2.rahtiapp.fi/). Feature requests and
+bug reports can be submitted
+[here](https://github.com/microbiome/miaDash/issues/).
```

Comment:
It could be appropriate to explicitly mention that it is hosted by Finnish IT Center for Science (CSC)